admin/app/controllers/admin/CommercialController.scala

```diff
@@ -74,13 +77,14 @@ class CommercialController(implicit context: ApplicationContext) extends Control
 
     val (hasNumericTestValue, hasStringTestValue) =
       lineItemsByAdTest partition { case (testValue, _) =>
-      def isNumber(s: String) = s forall Character.isDigit
+        def isNumber(s: String) = s forall Character.isDigit
```

Comment:
Good point - it looks like this is okay because:
a) the values used are the ad test values, which should never be empty and
b) it is just used to group the ad tests together when rendering the [ad test admin page](https://frontend.gutools.co.uk/commercial/adtests); worst case scenario the ad test with no name will be in the numeric section rather than the non-numeric.